src/logic/PlayerController.cpp

```diff
@@ -380,7 +380,7 @@ void PlayerController::unequipItem(Daedalus::GameState::ItemHandle item)
 
     EModelNode node = EModelNode::None;
 
-    if((itemData.flags & Daedalus::GEngineClasses::C_Item::ITM_CAT_EQUIPABLE) != Daedalus::GEngineClasses::C_Item::ITM_CAT_EQUIPABLE)
+    if((itemData.flags & Daedalus::GEngineClasses::C_Item::ITM_CAT_EQUIPABLE) != 0)
```

Comment:
Shouldn't it be `if((itemData.flags & Daedalus::GEngineClasses::C_Item::ITM_CAT_EQUIPABLE) == 0)`?
`flags & ITM_CAT_EQUIPABLE` masks out all flags except NF, FF, ..
As far as I understand you want to `return` when none of the flags are present.
Same goes for Line 460.